lib/live_view_native_swift_ui_charts/modifiers/view_fundamentals/chart_background.ex

```diff
@@ -0,0 +1,25 @@
+defmodule LiveViewNativeSwiftUiCharts.Modifiers.ChartBackground do
+  use LiveViewNativePlatform.Modifier
+
+  alias LiveViewNativeSwiftUi.Types.KeyName
+
+  modifier_schema "chart_background" do
+    field :alignment, Ecto.Enum,
+      values: ~w(
+      bottom
+      bottom_leading
+      bottom_trailing
+      center
+      leading
+      leading_last_text_baseline
+      top
+      top_leading
+      top_trailing
+      trailing
+      trailing_first_text_baseline
+    )a,
+      default: "center"
```

Comment:
Should this use an atom?
```suggestion
      default: :center
```